js/apps/admin-ui/cypress/e2e/realm_settings_general_tab_test.spec.ts

```diff
@@ -49,9 +49,12 @@ describe("Realm settings general tab tests", () => {
     sidebarPage.waitForPageLoad();
 
     // Disable realm
-    realmSettingsPage.toggleSwitch(`${realmName}-switch`);
+    const loadName = `load-disabled-${uuid()}`;
+    cy.intercept({ path: "/admin/realms/*", times: 1 }).as(loadName);
+    realmSettingsPage.toggleSwitch(`${realmName}-switch`, false);
```

Comment:
This could be nicer if the `toggleSwitch` had a functional arg that would perform arbitrary operation, `disableRealm` in this case. Leaving for the future since it could also open another can of worms